src/coreclr/src/gc/unix/gcenv.unix.cpp

```diff
@@ -851,6 +855,7 @@ static size_t GetLogicalProcessorCacheSizeFromOS()
             cacheSize = std::max(cacheSize, size);
     }
 
+#if defined(HOST_ARM64)
```

Comment:
left this ARM64 fallback in place for now, but this can be removed as well.

Reply:
It needs to stay for arm64. I have arm64 Odroid N2 with Ubuntu Linux and there are no `/sys/devices/system/cpu/cpu*/cache/*` entries present (in fact, none of the native tools that report cache sizes can report anything there).

Reply:
yeah I meant removing the `#if defined` and making it unconditional for all platforms.

Reply:
Ah, got it.

Reply:
I think leaving this ifdef in and using the conservative GC fallback is ok, for now at least.